dist/s_fixme

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+# Find FIXME comments related to the ticket potentially indicated by the branch name.
+# The script only fails if the branch name follows a specific syntax and FIXME comments are found,
+# otherwise, the script exits without failing.
+
+# Retrieve the current branch name.
+branch_name=$(git rev-parse --abbrev-ref HEAD)
+
+# We expect the following syntax (case insensitive): wt-<digits>[-<alphanum>].
+regex="^(wt|WT|wT|Wt)-[0-9]+(-[a-zA-Z0-9-]+)?"
+if [[ ! $branch_name =~ $regex ]]; then
+    exit 0
+fi
+
+# Get what could be the ticket id.
+ticket_id=$(echo "$branch_name" | cut -d "-" -f-2)
+
+# Check for fixme comments related to the ticket left behind.
+if grep -inr --exclude-dir=.git "fix.*$ticket_id" ../. > /dev/null 2>&1; then
```

Comment:
The comment at the beginning of the file says the script only fails if "FIXME comments are found".  So either the grep pattern here should include `fixme` instead of `fix`, or the comment should be updated.

Reply:
Thanks for spotting this discrepancy. I will update the comment as I have found "FIXME" and "FIX-ME" comments in the source code.